download_liblog.ps1

```diff
@@ -0,0 +1,4 @@
+$rootNamespace = "MyLib"
+$url = "https://raw.githubusercontent.com/damianh/LibLog/maybe-we-just-download-the-class/releases/latest/LibLog.cs"
+$response = Invoke-WebRequest $url
+$response.Content.Replace("`$rootnamespace`$", $rootNamespace) | Out-File -Encoding "UTF8" "LibLog.cs"
```

Comment:
The question is, how will library devs feel about sticking this in their project and running occasionally? 

Bit of a downgraded experience... not sure what other options I have.